library/std/src/sys/windows/c.rs

```diff
@@ -173,7 +173,7 @@ pub const PROGRESS_CONTINUE: DWORD = 0;
 
 pub const E_NOTIMPL: HRESULT = 0x80004001u32 as HRESULT;
 
-pub const INVALID_HANDLE_VALUE: HANDLE = !0 as HANDLE;
+pub const INVALID_HANDLE_VALUE: HANDLE = ptr::invalid_mut(!0);
```

Comment:
Would it be ever useful to have a method similar to `dangling` that returns `invalid(usize::MAX - align_of::<T>() + 1)`? I assume that there's some reason for not just using dangling or 2 * dangling here.

Reply:
I'm surprised to not be able to find official Microsoft documentation on `INVALID_HANDLE_VALUE` but it's one of their constants, not something Rust came up with (this is an FFI module).

Reply:
Oh, I see, it's just an FFI sentinel, not anything special. I saw it in a few places in the PR and was curious